### PR TITLE
ALBS-398: Added configurable setting logging_level

### PR DESCRIPTION
**Pull request by javihernandez**
Also, added some logging to release_planner.py

### CODE REVIEW

File: alws/app.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 from alws import routers
 from alws.test_scheduler import TestTaskScheduler
+from alws.config import settings
 
+import logging
```
Comment (anfimovdm) [inline alws/app.py:12]: `logging` should be importing in top level, not critical

File: alws/config.py [hunk 2]
```diff
@@ -36,5 +36,6 @@ class Settings(BaseSettings):
     sign_server_url: typing.Optional[str] = 'http://web_server:8000/api/v1/'
     sign_server_token: typing.Optional[str]
 
+    logging_level: typing.Optional[str] = 'INFO'
```
[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/config.py:39]: Right, I though about it, but I wasn't sure doing so. We are now expecting the user to enter the logging level through different means, I wanted to be consistent and not making the users confused about what they should use as the logging level when indicating it through entering an environmental variable directly, or by adding it to an vars.env file.
Does this make sense?

[the remark this answers is not in the document]
Comment (anfimovdm) [reply alws/config.py:39]: hmm, I didn't thought about this, I'm think it's make sense

[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/config.py:39]: Thanks @anfimovdm, resolving the conversation then ;)

**Review by Korulag — CHANGES_REQUESTED** [review]

File: alws/release_planner.py [hunk 3]
```diff
@@ -821,12 +831,14 @@ async def create_new_release(
             selectinload(models.Release.created_by),
             selectinload(models.Release.platform)
         ))
+        logging.info('New release %d successfully created', new_release.id)
```
[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/release_planner.py:834]: That's happening a few lines above, see https://github.com/AlmaLinux/albs-web-server/pull/237/files#diff-cbee670608d0c3f5e2c4645709fc22f396442128f2244febfdded169ca5d0b39R828

**Review by Korulag — APPROVED** [review]

File: alws/release_planner.py [hunk 4]
```diff
@@ -554,6 +561,7 @@ async def get_release_plan(
                             if repo['arch'] == pkg['arch']:
                                 repo['arch'] = weak_arch
                         beholder_cache[second_key] = replaced_pkg
+                        logging.debug('beholder_cache: %s', str(beholder_cache))
```
Comment (anfimovdm) [inline alws/release_planner.py:564]: this line will flood our logs, move it to 572 line, after `if not` condition

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/release_planner.py (hunk 4 above)
Comment (javihernandez) [reply alws/release_planner.py]: Good catch! Already fixed it. Sorry, I don't have a sense of how many times such loop can keep going :sweat_smile:

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**